com.ibm.wala.util/src/main/java/com/ibm/wala/util/graph/impl/BasicEdgeManager.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (c) 2002 - 2021 IBM Corporation.
```

Comment:
Why is this an appropriate copyright assignment and year range for brand-new code written by a non-IBM employee?